### PR TITLE
 [coteacher] Add primary teacher UI component

### PR DESCRIPTION
**Pull request by lfryemason**
## Warning!!

We have entered Pixel Lock for Hour of Code!

Computer Science Education Week will be happening from Dec 4 - Dec 10. Alongside this event, we will
be launching our new Hour of Code activity. Please consider any risk introduced in this PR that
could affect Dance Lab, instructions, saving and logging student progress, caching, or anything
related to the Hour of Code activities, old or new. Even small changes, such as a different button
color, are considered significant during this time. If this change will affect the new Hour of Code
activity in any way, join the morning change review to get your changes approved prior to merging.
Reach out to the Student Labs team for more details!

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dju90** [comment]
Do you have any screenshots of the changes?

**Comment by lfryemason** [comment]
image

File: apps/src/templates/sectionsRefresh/coteacherSettings/CoteacherSettings.jsx [hunk 1]
```diff
@@ -74,6 +76,13 @@ export default function CoteacherSettings({
   return (
     <div className={styles.expandedSection}>
       {i18n.coteacherAddInfo()}
+      <PrimaryTeacher
+        primaryTeacher={primaryTeacher}
+        numCoteachers={coteachers.length}
+      />
+      <label className={styles.label}>
```
Comment (etaderhold) [inline apps/src/templates/sectionsRefresh/coteacherSettings/CoteacherSettings.jsx:83]: Do we want this label in the case where we're not showing the primary teacher?

Comment (lfryemason) [reply apps/src/templates/sectionsRefresh/coteacherSettings/CoteacherSettings.jsx:83]: It's fine to leave it when the primary teacher is not shown: https://codedotorg.slack.com/archives/C0462BZKHL6/p1702482792484779

**Review by etaderhold — APPROVED** [review]

**Pull request merged**